easybuild/easyconfigs/g/GSettings-Desktop-Schemas/GSettings-Desktop-Schemas-3.34.0-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'MesonNinja'
+
+name = 'GSettings-Desktop-Schemas'
+version = '3.34.0'
+
+homepage = 'https://gitlab.gnome.org/GNOME'
```

Comment:
@Flamefire this page is probably more appropriate?
https://gitlab.gnome.org/GNOME/gsettings-desktop-schemas

Reply:
Oh you are right, just copied half of it :/

Reply:
Ah, I got the URL from https://github.com/GNOME/gsettings-desktop-schemas